modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/meta.yml

```diff
@@ -0,0 +1,31 @@
+name: "atlasgeneannotationmanipulation_gtf2featureannotation"
+description: Generate tables of feature metadata from GTF files
+keywords:
+  - gtf
+  - annotation
+tools:
+  - "atlasgeneannotationmanipulation":
+      description: "Scripts for manipulating gene annotation"
+      homepage: "https://github.com/ebi-gene-expression-group/atlas-gene-annotation-manipulation"
+      documentation: "https://github.com/ebi-gene-expression-group/atlas-gene-annotation-manipulation"
+      tool_dev_url: "https://github.com/ebi-gene-expression-group/atlas-gene-annotation-manipulation"
+      licence: "['Apache-2.0']"
+
+input:
+  - gtf:
```

Comment:
I suggest adding information about the meta map as well. [ex](https://github.com/nf-core/modules/blob/master/modules/nf-core/abacas/meta.yml)